cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java

```diff
@@ -18,79 +18,78 @@
 import com.cst438.domain.CourseRepository;
 
 @RestController
-@CrossOrigin 
+@CrossOrigin
 public class AssignmentController {
-	
-	@Autowired
-	AssignmentRepository assignmentRepository;
-	
-	@Autowired
-	CourseRepository courseRepository;
-	
-	@GetMapping("/assignment")
-	public AssignmentDTO[] getAllAssignmentsForInstructor() {
-		// get all assignments for this instructor
-		String instructorEmail = "dwisneski@csumb.edu";  // user name (should be instructor's email) 
-		List<Assignment> assignments = assignmentRepository.findByEmail(instructorEmail);
-		AssignmentDTO[] result = new AssignmentDTO[assignments.size()];
-		for (int i=0; i<assignments.size(); i++) {
-			Assignment as = assignments.get(i);
-			AssignmentDTO dto = new AssignmentDTO(
-					as.getId(), 
-					as.getName(), 
-					as.getDueDate().toString(), 
-					as.getCourse().getTitle(), 
-					as.getCourse().getCourse_id());
-			result[i]=dto;
-		}
-		return result;
-	}
-	
-	// TODO create CRUD methods for Assignment
-
-	@GetMapping("/assignment/{assignment_id}")
-
-	public AssignmentDTO getAssignment(@PathVariable("assignment_id") int assignment_id){
-		Assignment assignment = assignmentRepository.findById(assignment_id)
-				.orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Assignment not found"));
-		return new AssignmentDTO(
-				assignment.getId(),
-				assignment.getName(),
-				assignment.getDueDate().toString(),
-				assignment.getCourse().getTitle(),
-				assignment.getCourse().getCourse_id()
-		);
+
+    @Autowired
+    AssignmentRepository assignmentRepository;
+
+    @Autowired
+    CourseRepository courseRepository;
+
+    @GetMapping("/assignment")
+    public AssignmentDTO[] getAllAssignmentsForInstructor() {
+        // get all assignments for this instructor
+        String instructorEmail = "dwisneski@csumb.edu";  // user name (should be instructor's email)
+        List<Assignment> assignments = assignmentRepository.findByEmail(instructorEmail);
+        AssignmentDTO[] result = new AssignmentDTO[assignments.size()];
+        for (int i = 0; i < assignments.size(); i++) {
+            Assignment as = assignments.get(i);
+            AssignmentDTO dto = new AssignmentDTO(
+                    as.getId(),
+                    as.getName(),
+                    as.getDueDate().toString(),
+                    as.getCourse().getTitle(),
+                    as.getCourse().getCourse_id());
+            result[i] = dto;
+        }
+        return result;
```

Comment:
DTO objects are preferable in rest api since they won't return potential relationships, so good job following the convention to transform objects into records so avoid possible issues